app/uk/gov/hmrc/fhregistrationfrontend/controllers/BusinessPartnersConfirmAddressController.scala

```diff
@@ -44,18 +45,18 @@ class BusinessPartnersConfirmAddressController @Inject()(
   )
 
   val partnerName = "test business partner"
-  val postAction: Call = routes.BusinessPartnersConfirmAddressController.next()
   val backLink = routes.BusinessPartnerAddressController.load().url
 
-  def load(): Action[AnyContent] = userAction { implicit request =>
+  def load(index: Int, mode: Mode = NormalMode): Action[AnyContent] = userAction { implicit request =>
+    val postAction: Call = routes.BusinessPartnersConfirmAddressController.next(index, mode)
     if (config.newBusinessPartnerPagesEnabled) {
       Ok(view.business_partners_confirm_partner_address(address, partnerName, postAction, backLink))
     } else {
       errorHandler.errorResultsPages(Results.NotFound)
     }
   }
 
-  def next(): Action[AnyContent] = userAction { implicit request =>
+  def next(index: Int, mode: Mode = NormalMode): Action[AnyContent] = userAction { implicit request =>
```

Comment:
You need to update to use the new action. Even though you won't require to read or write from useranswers you will still need them to be present to access the page. The dataRequiredAction also includes the "newBusinessPartnerPagesEnabled" logic so that can be removed from here too